test/unittests/libstorage/test_MemoryTable2.cpp

```diff
@@ -0,0 +1,105 @@
+/*
+ * @CopyRight:
+ * FISCO-BCOS is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * FISCO-BCOS is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with FISCO-BCOS.  If not, see <http://www.gnu.org/licenses/>
+ * (c) 2016-2018 fisco-dev contributors.
+ */
+
+#include "MemoryStorage2.h"
+#include <libdevcore/FixedHash.h>
+#include <libstorage/Common.h>
+#include <libstorage/MemoryTable2.h>
+#include <libstorage/Storage.h>
+#include <libstorage/Table.h>
+#include <tbb/parallel_for.h>
+#include <boost/test/unit_test.hpp>
+#include <string>
+#include <thread>
+#include <vector>
+
+using namespace std;
+using namespace dev;
+using namespace dev::storage;
+
+namespace test_MemoryTableFactory2
+{
+struct MemoryTable2Fixture
+{
+    MemoryTable2Fixture()
+    {
+        auto storage = std::make_shared<MemoryStorage2>();
+        m_table = std::make_shared<MemoryTable2>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'm_table' is assigned in constructor body. Consider performing initialization in initialization list.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5502559)